apps/web/src/components/layout/components/v2/BridgeUpdateModal.tsx

```diff
@@ -78,7 +78,7 @@ export const BridgeUpdateModal: FC<BridgeUpdateModalProps> = ({ isOpen, toggleOp
           result = await validateBridgeUrl({ bridgeUrl: url });
         }
         if (!result.isValid) {
-          throw new Error('The provided URL is not the Novu Endpoint URL');
+          throw new Error(result.error);
```

Comment:
This small change in Web now ensures that the actual Bridge error message is shown to the user.